extensions/tasks/NLUTestV0/publishTestResults.ts

```diff
@@ -0,0 +1,50 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+
+type TestRunner = "JUnit" | "NUnit" | "VSTest" | "XUnit" | "CTest";
+
+const TESTRUN_SYSTEM = 'VSTS - PTR';
```

Comment:
can you add a comment about what this constant does (i.e., what's a testRunSystem)?